### PR TITLE
(DOCSP-34509) Adds release notes, source constant, submodule updates for v1.13.0

### PR DESCRIPTION
**Pull request by sarahsimpers**
[Jira](https://jira.mongodb.org/browse/DOCSP-34509)
[Staging](https://preview-mongodbsarahsimpers.gatsbyjs.io/atlas-cli/DOCSP-34509/atlas-cli-changelog/#atlas-cli-1.13.0)
[Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6557822f5b686d6bd950d331)

### CODE REVIEW

File: source/atlas-cli-changelog.txt [hunk 1]
```diff
@@ -14,6 +14,37 @@
...
+  
+  - :ref:`atlas-clusters-search-indexes-list`
+  - :ref:`atlas-clusters-search-indexes-describe`
+  - :ref:`atlas-deployments-search-indexes-list`
+  - :ref:`atlas-deployments-search-indexes-describe`
+- Adds the ``--expireAfterDays`` option for
+  :ref:`atlas-clusters-onlineArchives-create` and
+  :ref:`atlas-clusters-onlineArchives-update` commands.
+- Improved the error message for the :ref:`atlas-deployments-list`
+  command if you aren't authenticated.
```
Comment (corryroot) [inline source/atlas-cli-changelog.txt:39]:
```suggestion
- Improves the error message for the :ref:`atlas-deployments-list`
  command if you aren't authenticated.
```

File: source/atlas-cli-changelog.txt [hunk 2]
```diff
@@ -14,6 +14,37 @@
...
+- Fixes a bug that prevented organization fetching, which prevented
+  some users from logging in.
+- Fixes a bug where :ref:`atlas-logs-download` didn't respect the
+  ``--start`` and ``--end`` flags.
+- Fixes a local development bug when you create an index from a file.
```
Comment (corryroot) [inline source/atlas-cli-changelog.txt:44]:
```suggestion
- Fixes a local deployment bug when you create an index from a file.
```

**Review by corryroot — APPROVED** [review]
LGTM! One nit and one possible fix.

**Review by blva — APPROVED** [review]

**Comment by blva** [comment]
cc @tibulca and @fmenezes in case we're missing any fixes in the description

**Review by tibulca — APPROVED** [review]

**Review by fmenezes — APPROVED** [review]

**Pull request merged**